model-import/cli-import.mdx

```diff
@@ -1,300 +1,172 @@
-
-* `output.json`This file will have the `name` of the output key that the `def infer` the function is going to return.
-
-Update the input.json / Output Json as per your model. 
-
-```
-// Input.json 
-{
-  "inputs": [
-    {
-      "data": [
-        "Image of horse near beach"
-      ],
-      "name": "prompt",
-      "shape": [
-        1
-      ],
-      "datatype": "BYTES"
-    }
-  ]
-}
-```
-
-```
-// Output.json 
-
+    - "accelerate==0.33.0"
+    - "torch==2.4.0"
+    - "transformers==4.44.0"
+    - "diffusers==0.30.0"
+    ```
+
+2. Run the below command to create the runtime
+    
+    ```bash
+    inferless runtime create --name <runtime_name> --path ./inferless_runtime_config.yaml
+    ```
+
+3. Deploy with runtime
+    
+    ```bash
+    inferless deploy --gpu t4 --region <regionname> --runtime <runtimename> 
+    ```
+
+### Volumes
+
+Volumes in Inferless are NFS-like writable storage spaces that can be connected to multiple replicas simultaneously. They serve several key purposes:
+
+1. Storing model parameters
+2. Archiving datasets (similar to centralized storage)
+3. Setting up shared caches for collaborative tasks
+
+### Creating and uploading weights
+
+1. To create a new volume. Run this command
+    
+    ```bash
+    inferless volume create --name <volumename>
+    ```
+    
+2. New volume will be created and you will be shown the `infer_path` where you can store the weights. Make sure you keep this handy
+3. Once the volume is created. Upload the weights using this command and paste the `infer_path` in the destination
+    
+    ```bash
+    inferless volume cp --source <source_path> --destination <remote_path (Infer path)>
+    ```
+    
+4. Your files will be copied to the server and your volume is ready to be used.
+5. To use these weights, in `app.py` you can specify the mount path from where the weights can be accessed (Eg: `/var/nfs-mount/<volume_name>`)
+    
+    ```json
+    from diffusers import StableDiffusionPipeline
+    import torch
+    from io import BytesIO
+    import base64
+    
+    MODEL_WEIGHTS_DIR =  "/var/nfs-mount/my_volume"
+    
+    class InferlessPythonModel:    
+        def initialize(self):
+            self.pipe = StableDiffusionPipeline.from_pretrained(
+                "/var/nfs-mount/my_volume",
+                use_safetensors=True,
+                torch_dtype=torch.float16,
+                device_map='auto'
+                local_files_only=True
+            )
+    ```
+    
+6. Run the command to create a new model using the above model weights 
+    
+    ```bash
+    inferless init --name <modelname>
+    ```
+    
+7. Once the model is initialized. Deploy it using this command (Make sure the path defined inside `app.py` and here in the command should be same)
+    
+    ```bash
+    inferless deploy --gpu t4 --region <regionname> --volume <volumename> --mountpath mount_path 
```

Comment:
Command is incorrect

Reply:
@Kailashinfer  inferless deploy --gpu t4 --region <region_name> --volume <volume_name> --mountpath <mount_path>